.bazelrc

```diff
@@ -136,6 +136,7 @@ build --host_java_toolchain=//third_party/toolchains/java:tf_java_toolchain
 # environment variable "TF_MKL_ROOT" every time before build.
 build:mkl --define=build_with_mkl=true --define=enable_mkl=true
 build:mkl --define=tensorflow_mkldnn_contraction_kernel=0
+build:mkl --define=build_with_mkl_dnn_v1_only=true
 build:mkl -c opt
 
 # This config option is used to enable MKL-DNN open source library only,
```

Comment:
Should we make `mkl_open_source_only` use v1.2 too?

Reply:
That's a good point, Penporn. If we add v1.2 to that config option, then it would enable v1.2 always (as a default in mkl config and in open_source_only version). Currently, v0.21.3 is a default version if one uses `mkl_open_source_only` option. If we add v1.2 to that option, then there would not be a way to build with v0.21.3 (if one needs to). Also, if we add v1.2 to that option, then we should also clean up all of v0.21.3 support (which is not there in this PR). So better to leave this option as it is, and we will clean it up later when we are removing complete support for v0.21.3. Does that sound reasonable?

Reply:
That's a good point. Let's keep it this way then. Thank you for the explanation! :)

Reply:
OK, thanks!